src/lib/transition_frontier/catchup_tree.ml

```diff
@@ -1,31 +0,0 @@
-(* This is temporary until we decide to switch over fully to "super catchup".
```

Comment:
I think you meant do move this file instead of deleting it

Reply:
yes i did